src/components/Shared/SimpleMessagePopup.tsx

```diff
@@ -15,6 +15,7 @@ const mapDispatchToProps = {
 
 export enum EnumButtonSpec {
   Ok = 1,
+  HIDE = 2
```

Comment:
Can you please replace all the changes in this file with the following:

```
import * as uiActions from "actions/uiActions";
import ModalPopup from "./ModalPopup";
import * as React from "react";
import * as css from "./SimpleMessagePopup.scss";
import { connect } from "react-redux";
import { IRootState } from "reducers";

interface IDispatchProps {
  hideSimpleMessage: typeof uiActions.hideSimpleMessage;
}

const mapDispatchToProps = {
  hideSimpleMessage: uiActions.hideSimpleMessage,
};

export enum EnumButtonSpec {
  Ok = 1,
  // eventually will turn this into a confirmation modal, adding OkCancel, Yes/No and stuff
}

export interface ISimpleMessagePopupProps {
  closeHandler?: (event: any) => void;
  body: string | JSX.Element;
  buttonSpec?: EnumButtonSpec;
  /**
   * `buttonSpec` is ignored if `hideFooter` is true
   */
  hideFooter?: boolean;
  title?: string | JSX.Element;
}

interface IStateProps {
  options: ISimpleMessagePopupProps;
  showing: boolean;
}

const mapStateToProps = (state: IRootState): IStateProps => {
  return {
    options: state.ui.simpleMessageOptions,
    showing: state.ui.simpleMessageOpen,
  };
};

class SimpleMessagePopup extends React.Component<IDispatchProps & IStateProps, null> {

  private closeHandler = (event: any) => {
    this.props.hideSimpleMessage();
    if (this.props.options.closeHandler) {
      this.props.options.closeHandler(event);
    }
  }

  private renderButtons = (): JSX.Element => {
    const okButton = (<button className={css.closeButton} onClick={this.closeHandler}>OK</button>);
    switch (this.props.options.buttonSpec) {
      case EnumButtonSpec.Ok:
      default:
        return okButton;
    }
  }

  public render(): RenderOutput {

    if (!this.props.showing) {
      return "";
    }

    if (!this.props.options.body) {
      throw new Error("message body is required");
    }

    return (
      <div className={css.modalContainer}>
        <ModalPopup
          closeHandler={this.closeHandler}
          header={
            <div className={css.modalHeader}>
              <div className={css.title}>{this.props.options.title ?? "Alchemy"}</div>
              <div className={css.closeButtonX} onClick={this.closeHandler}><img src={"/assets/images/Icon/close-grey.svg"} />
              </div>
            </div>
          }
          body={
            <div className={css.modalBody}>{this.props.options.body}</div>
          }
          footer={
            this.props.options.hideFooter ?
              <div className={css.modalFooter}>{this.renderButtons()}</div>
              : undefined
          }
        />
      </div>
    );
  }
}

export default connect(mapStateToProps, mapDispatchToProps)(SimpleMessagePopup);
```

Reply:
This will basically remove my changes. The "OK" button being displayed in the modal is something I brought up early on and was told to go ahead and make the change. I didn't want both an "OK" button and "Send" button displayed in the modal. The user can click away from the modal or click the 'X' close in the popup to get rid of it.

Reply:
The change is actually saying that if "hideFooter" is set, it will show the button. I think you want that reversed, correct?

Reply:
> > should there be an "envelope" here ? I do not see that .
> > here is the review app https://alchemy-dev-mradczgvxpkcjdiuxh.herokuapp.com/daos/
> > @mdcoon
> > <img alt="Screen Shot 2020-06-17 at 17 22 59" width="561" src="https://user-images.githubusercontent.com/5994398/84910225-4b7dd000-b0bf-11ea-83e7-f940454e6bb7.png">
> 
> Yes. I did a push this morning to the BUIDLHub fork that was missing assets. I wonder if it's not pulled into the PR.

it was a review app issue . 
the envelop showing up now ! https://alchemy-dev-oxyfc3mtejjbgoiq9d.herokuapp.com/

Reply:
> The change is actually saying that if "hideFooter" is set, it will show the button. I think you want that reversed, correct?

Oops, sorry, lost that in an edit.  Yes please, reverse the logic.